src/main/java/org/apache/maven/shared/dependency/analyzer/ClassFileVisitorUtils.java

```diff
@@ -99,7 +97,8 @@ private static void acceptJar( URL url, ClassFileVisitor visitor )
                 // ignore files like package-info.class and module-info.class
                 if ( name.endsWith( ".class" ) && name.indexOf( '-' ) == -1 )
                 {
-                    visitClass( name, in, visitor );
+                    // Even on Windows Jars use / as the separator character
```

Comment:
The comment is redundant and wrong because this has nothing to do with Windows. REad the PKZIP implementation note: separators are always foward slashes regarding of the OS.

Reply:
Yes, that's the point. The previous version was using the system dependent line separator, which was why the test was failing on Windows but not Linux.

Reply:
comment was changed